### PR TITLE
Clean up History copy

### LINKED ISSUE #57: Clarify that history is 100% local and not shared
Re-reading the current blurb on the history page, it's not immediately clear that this data is generated and stored locally, and never shared. We could rephrase it to make it a bit more clear.

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/65095222-clarify-that-history-is-100-local-and-not-shared?utm_campaign=plugin&utm_content=tracker%2F60754889&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F60754889&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by cassidyjames**
Fixes #57, along with some other cleaning up for clarity. Open to input, but I think it reads a lot better.

### CODE REVIEW

File: src/Utilities.vala [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+*
+* You should have received a copy of the GNU General Public
+* License along with this program; if not, write to the
+* Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+* Boston, MA 02110-1301 USA
+*/
+
+namespace SecurityPrivacy.Utilities {
+    private static string get_operating_system_name () {
+        string system = _("Your system");
```
Comment (danrabbit) [inline src/Utilities.vala]: We used "This" instead of "Your" because 2nd person often causes some confusion for translators about formality

File: src/Utilities.vala [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+* General Public License for more details.
+*
+* You should have received a copy of the GNU General Public
+* License along with this program; if not, write to the
+* Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+* Boston, MA 02110-1301 USA
+*/
+
+namespace SecurityPrivacy.Utilities {
```
Comment (danrabbit) [inline src/Utilities.vala]: I'm not sure it makes sense to move this into its own class if it's only used in the one class. Unless there is another class here using this method?

Comment (cassidyjames) [reply src/Utilities.vala]: The problem I had is that I could not use it inside the `Object ()`; I was just getting `null`. It's possible I needed to do something other than move it to its own class, though.

Comment (danrabbit) [reply src/Utilities.vala]: If you just make that method `static` then it works :)

**Comment by cassidyjames** [comment]
@danrabbit review addressed

File: src/Views/TrackPanel.vala [hunk 3]
```diff
@@ -24,17 +24,20 @@ public class SecurityPrivacy.TrackPanel : Granite.SimpleSettingsPage {
     private Widgets.ClearUsagePopover remove_popover;
 
     public TrackPanel () {
-        Object (activatable: true,
-                description: _("This operating system can gather useful statistics about file and app usage to provide extra functionality. If other people can see or access your account, you may wish to limit which items are recorded."),
-                icon_name: "document-open-recent",
-                title: _("History"));
+        Object (
+            activatable: true,
+            description: (_("%s can store local usage data to provide extra functionality like offering recently-used files and more relevant local search. Regardless of this setting, usage data is never transmitted off of this device or to third parties.").printf (get_operating_system_name ())),
```
Comment (tintou) [inline src/Views/TrackPanel.vala]: It's long enough, don't include another parenthesis
```suggestion
            description: _("%s can store local usage data to provide extra functionality like offering recently-used files and more relevant local search. Regardless of this setting, usage data is never transmitted off of this device or to third parties.").printf (get_operating_system_name ()),
```

Comment (cassidyjames) [reply src/Views/TrackPanel.vala]: Derp, thanks.
